package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "webpack serve --open"
```

Comment:
- [ ] Please, add the next option here `"dev": "webpack --mode development"` as it will help you to run your project in development server.